.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+##### Description of Change
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Contributors guide: https://github.com/zephyrproject-rtos/zephyr/blob/master/CONTRIBUTING.rst
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included and stakeholders cc'd
+- [ ] `sanitycheck` passes
+- [ ] tests are [changed or added]
+- [ ] relevant documentation is changed or added
+
+
+##### Release Notes
+<!-- Used to describe release notes for future release versions. -->
+
+Notes: <!-- One-line Change Summary Here-->
```

Comment:
one-line change summary at the end?  That's new.

Reply:
this is for the release notes.